src/qt/transactionview.cpp

```diff
@@ -395,6 +396,8 @@ void TransactionView::contextualMenu(const QPoint &point)
     hash.SetHex(selection.at(0).data(TransactionTableModel::TxHashRole).toString().toStdString());
     abandonAction->setEnabled(model->wallet().transactionCanBeAbandoned(hash));
     bumpFeeAction->setEnabled(model->wallet().transactionCanBeBumped(hash));
+    copyAddressAction->setEnabled(GUIUtil::hasEntryData(transactionView, 0, TransactionTableModel::AddressRole));
```

Comment:
How to test this line of the code?

Reply:
I tested with some JoinMarket transactions, where I had partially imported JM wallet addresses as watchonly (some of inputs belonged to the wallet, but no output addresses).